packages/manager/apps/web/client/app/email-domain/order/email-domain-order.service.js

```diff
@@ -1,67 +1,33 @@
-import assign from 'lodash/assign';
-import map from 'lodash/map';
+import { CONFIGURATION_LABEL, GENERIC_PRODUCT, PRODUCT_NAME } from './order.constants';
 
-angular.module('services').service(
-  'MXPlan',
-  class MXPlan {
-    /**
-     * Constructor
-     * @param $q
-     * @param OvhHttp
-     */
-    constructor($q, OvhHttp) {
-      this.$q = $q;
-      this.OvhHttp = OvhHttp;
-    }
+export default class MXPlan {
+  /* @ngInject */
+  constructor($q, OrderService) {
```

Comment:
so wucOrderCartService?

Reply:
@frenautvh A refactor will be handled in another PR (here master is the target not `develop`

Reply:
https://github.com/ovh/manager/pull/2013